src/Pelagos/Util/Search.php

```diff
@@ -32,6 +32,26 @@ class Search
      */
     protected $entityManager;
 
+    /**
+     * Elastic index title field.
+     */
+    const INDEX_FIELD_TITLE = 'title';
```

Comment:
Any suggestions in naming these constants better? not really happy with them

Reply:
An array in a constant could work. Then just assign it a named (key/pair) array.